### DISCUSSION TITLE
lockFileMaintenance: group by lockfile type

### DISCUSSION THREAD

**Discussion by katexochen** [renovatebot/renovate#28068] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

I have a project with multiple lock files (flake.lock, yarn.lock). I'd like renovate to update those lock files independently from each other (in separate PRs). Is there a way this can be archived? 

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
This might work:

```
{
  "packageRules": [{
    "matchUpdateTypes": ["lockFileMaintenance"],
    "additionalBranchPrefix": "{{manager}}-"
  }]
}
```

**Reply by katexochen** [reply]
Thanks, works indeed! :)
